meta-luneos/recipes-core/systemd/systemd/0001-systemd-oomd-depend-on-swap-on.patch

```diff
@@ -0,0 +1,23 @@
+From a23bcc49098ce6fb92192367d0d074618ed0c97c Mon Sep 17 00:00:00 2001
+From: Hotaek Jung <hotaek.jung@lge.com>
+Date: Mon, 12 Jul 2021 19:16:26 +0900
+Subject: [PATCH] systemd-oomd: depend on swap-on
+
+Upstream-Status: Pending
+---
+ units/systemd-oomd.service.in | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/units/systemd-oomd.service.in b/units/systemd-oomd.service.in
+index 82bd6245f8..94b1b0c786 100644
+--- a/units/systemd-oomd.service.in
++++ b/units/systemd-oomd.service.in
+@@ -13,6 +13,8 @@ Documentation=man:systemd-oomd.service(8)
+ Documentation=man:org.freedesktop.oom1(5)
+ DefaultDependencies=no
+ Before=multi-user.target shutdown.target
++Wants=swap-on.service
++After=swap-on.service
```

Comment:
I don't really see the logic here, but I don't think it will affect us anyway, so ok

Reply:
This comes 1:1 from OSE... Also not sure, trial and error. Anyway this will solve some runtime errors. Just trying to fix some of the low-hanging fruit like this.